### ISSUE TITLE
Welcome to Study Groups, @colleenboodleman !

### ISSUE THREAD

**Issue by auremoser** [colleenboodleman/Womens-GeoViz-StudyGroup#3]
Hi @colleenboodleman !
Thanks for your interest in the [Mozilla Science Lab](https://mozillascience.org/), it's great to hear you're building a [Study Group](https://github.com/mozillascience/studygroup).

My name is [Aurelia](https://mozillascience.org/u/auremoser) and I work at the Mozilla Science Lab, a group building projects/programs globally to support open research and citizen science, including the Study Groups project. Just wanted to introduce myself as a resource, and answer any questions you might have.

Feel free to:
* search for ideas in [the handbook](http://mozillascience.github.io/studyGroupHandbook/)
* build out [a site for your Study Group](https://github.com/mozillascience/studyGroup) and cull from our [library of lessons](https://github.com/mozillascience/studyGroupLessons) for early sessions
* join our monthly study group call (first Fridays, 11AM EST), [subscribe to our mailing list to be notified of those calls](https://mail.mozilla.org/listinfo/msl-study-groups)
* [subscribe to our mailing list](https://mail.mozilla.org/listinfo/msl-study-groups)
* follow our #mozstudy hashtag on twitter, or [our list](https://twitter.com/MozillaScience/lists/mozstudy), ping [@auremoser](https://twitter.com/MozillaScience) to be added

**Join us on our next Study Group Learning call, Friday June 17th, 11am EST: https://public.etherpad-mozilla.org/p/mozilla-studygroup-leads-jun-2016**.

Email me at [aurelia@mozillafoundation.org](mailto:aurelia@mozillafoundation.org) with questions, or contact/social media information to join our twitter community and Study Group email listserv. Would love to learn about your lessons, challenges, topic of interest, and events. 

Thanks for reading!

Aurelia
Mozilla Science Lab
[@MozillaScience](https://twitter.com/MozillaScience)

**Comment by colleenboodleman** [comment]
Hi Aurelia, 
Please add the Women's GeoViz Study Group, S.F. California, Hackbright Academy to the study groups map at https://science.mozilla.org/programs/studygroups as part of the US study groups. I tried but was unable to add the study group to the mozilla map.  Thanks!

**Comment by auremoser** [comment]
Added! Thanks so much @colleenboodleman !

![screen shot 2016-06-08 at 10 40 40 am](https://cloud.githubusercontent.com/assets/1559703/15898041/b213f4f8-2d64-11e6-80be-54ef48a2d0d1.png)

**Comment by colleenboodleman** [comment]
@auremoser 
hi, id like to modify the "who we are" section on the http://colleenboodleman.github.io/Womens-GeoViz-StudyGroup/ website, is there a place in the yaml file or html to do this?? thanks, Colleen

**Comment by auremoser** [comment]
Yes @colleenboodleman, it's in the `members.yml` file which is here in your repository: [Womens-GeoViz-StudyGroup/_data/members.yml](https://github.com/colleenboodleman/Womens-GeoViz-StudyGroup/blob/gh-pages/_data/members.yml).

For example, editing my `members.yml` for [this study group fork](https://github.com/ScienceHackSunday/studyGroup): 

![screen shot 2016-06-29 at 6 55 18 pm](https://cloud.githubusercontent.com/assets/1559703/16471518/3b1282e8-3e2b-11e6-87b8-3f7206a297ef.png)

Produced [this site](http://sciencehacksunday.nyc/):

![screen shot 2016-06-29 at 6 57 29 pm](https://cloud.githubusercontent.com/assets/1559703/16471522/402b852c-3e2b-11e6-841c-26edd40101a5.png)

Hope that helps, and thank you for asking; please let me know if you have any other troubles!
